_plugins/jekyll_serach_fixer.rb

```diff
@@ -0,0 +1,14 @@
+module Jekyll
```

Comment:
is this filename right?

Reply:
Thanks for the catch - updating the file name!